### PR TITLE
Edit feature

### PR DESCRIPTION
**Pull request by Iamayushgupta**

### CODE REVIEW

File: src/routers/user.js [hunk 1]
```diff
@@ -79,8 +79,19 @@ router.get("/signup", async (req, res) => {
 router.get("/users", async (req, res) => {
     try {
         const city = req.query.city
-        const restaurants = await User.find({city})
-        res.render('searchByCity', { restaurants })
+        const name = req.query.name
+
+        if (!city){
+            var restaurants = await User.find({ name })
```
Comment (abhigupta4) [inline src/routers/user.js:85]: What if name is also empty?

Comment (Iamayushgupta) [reply src/routers/user.js:85]: It wont show anything anyways

Comment (abhigupta4) [reply src/routers/user.js:85]: Yes but it should be explicity handled. These cases should be handled properly

**Review by abhigupta4 — APPROVED** [review]
Approving but make changes in next PR

**Pull request merged**